feature/bgp/bmp/otg_tests/bmp_base_session_test/README.md

```diff
@@ -0,0 +1,159 @@
+#        }
+#      }
+#    }
+#  }
+#}
+
+The below yaml defines the OC paths intended to be covered by this test.
+
+
+#TODO paths in flat format:
+  ## Config paths
+# /network-instances/network-instance/protocols/protocol/bgp/global/bmp/config/enabled:
+# /network-instances/network-instance/protocols/protocol/bgp/global/bmp/config/connection-mode:
+# /network-instances/network-instance/protocols/protocol/bgp/global/bmp/config/local-address:
+# /network-instances/network-instance/protocols/protocol/bgp/global/bmp/config/statistics-timeout:
+# /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/config/name:
+# /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/config/address:
+# /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/config/port:
+# /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/config/policy-type:
+# /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/config/exclude-non-eligible:
+
+
+  ## state paths
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/state/enabled:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/state/connection-mode:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/state/local-address:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/state/statistics-timeout:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/state/idle-time:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/state/probe-count:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/state/probe-interval:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/state/address:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/state/port:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/state/connection-status:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/state/uptime:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/state/flap-count:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/state/policy-type:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/state/exclude-non-eligible:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/state/message-counters/total:
+ # /network-instances/network-instance/protocols/protocol/bgp/global/bmp/stations/station/state/message-counters/statistics:
+
+rpcs:
+  gnmi:
+    gNMI.Set:
+      union_replace: true
+      replace: true
+    gNMI.Subscribe:
+```
+
+## Minimum DUT platform requirement
+
+*   FFF - Fixed Form Factor
+*   MFF - Modular Form Factor
```

Comment:
vRX - Virtual Router. (bmp is control plane only, no hardware specific feature is being exercised here IMO)